metadata_controller.py

```
@@ -221,8 +221,14 @@ def move_to_publisher_folder(self, new_name, publisher_int):
                     dir_path = os.path.join(dirpath, dirname)
                     new_path = os.path.join(dir_path, new_name)
                     shutil.move(self.original_filepath, new_path)
+                    new_path = Path(new_path)
                     print(f"[INFO] Moved file to {dir_path}")
-                    self.inputter.insert_filepath(new_path)
+                    try:
+                        relative_path = new_path.relative_to(ROOT_DIR)
+                        self.inputter.insert_filepath(relative_path)
+                    except ValueError as e:
```

Ruff (0.14.4)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)